url_example_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Walter Scheper <walter.scheper@gmail.com>
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package rapid_test
+
+import (
+	"fmt"
+
+	"pgregory.net/rapid"
+)
+
+func ExampleDomain() {
+	gen := rapid.Domain()
+
+	for i := 0; i < 5; i++ {
+		fmt.Println(gen.Example(i))
+	}
+
+	// Output:
+	// MV2zb0-S2j.trAveLcHAnnEL
+	// Z.CU
+	// r.ABBotT
+	// r.AcCoUNTaNT
+	// R.
```

Comment:
`R.` is probably not what we wanted to generate?

Reply:
Yeah, that's probably coming from the two empty string entries that are incorrectly included.